gitops/gitops.go

```diff
@@ -21,6 +21,7 @@ import (
 	"path/filepath"
```

Comment:
Does this file (gitops.go) not need to be extracted out too?

Reply:
So the interactions with git will remain within HAS?

Reply:
Yes, it will, as the [issue](https://issues.redhat.com/browse/PLNSRVCE-138) say.